### ISSUE TITLE
Proposal : add an option to opt out of automated url delegation

### ISSUE THREAD

**Issue by gbouteiller** [ascorbic/unpic#119]
Hello,
I would like to do the following :
- host my images on Imagekit
- host my website on Vercel
- use Vercel Image optimization
For this to happen, an option should be created to opt out of url delegation. 
I can make a PR if needed.
Thank you

**Comment by ascorbic** [comment]
I'm trying to work out the scenario here. Can you explain why you'd want to use Vercel to optimize your Imagekit source images?

**Comment by gbouteiller** [comment]
Yes, of course. Imagekit can be used (like Cloudinary) for image optimization and/or storage. For Imagekit, depending of the plan you choose, you have a number of Gb for your storage and another for the bandwidth. Vercel has its own pricing regarding image optimization. For Cloudinary you have credits that can be either used for bandwidth or storage. It can be interesting to have the possibility to choose which service stores and which service optimizes.
A concrete example : you want to stay on the free tier of Cloudinary but you have 25Gb of images (so 25 credits used) and you need some image optimization. So if your website is hosted on Vercel, you can opt out of url delegation to let Vercel manage the optimization while all your Cloudinary credits are used for storage only.

**Comment by ascorbic** [comment]
OK, got it. This would want to be in the cdn options for the ones in question

**Comment by gbouteiller** [comment]
@ascorbic So would you prefer to integrate it or do you want me to make a PR?
